rosdep/base.yaml

```
@@ -4026,6 +4026,11 @@ nkf:
   fedora: [nkf]
   gentoo: [app-i18n/nkf]
   ubuntu: [nkf]
+nmap:
+  debian: [nmap]
```

```suggestion
  archlinux: [nmap]
  debian: [nmap]
```
Suggesting the archlinux add while I'm here: https://www.archlinux.org/packages/extra/x86_64/nmap/

---

@Vuwij could you accept this change suggestion then we can merge this?

---

> @Vuwij could you accept this change suggestion then we can merge this?

In order to merge the change I've committed the addition of the Arch key directly to the PR branch and am taking the above comment as approval of the final state.